*** auto-tests/A-16-06-M-ok.out	2020-11-26 19:26:04.100000000 +0000
--- auto-tests/A-16-06-M-ok.outhyp	2020-11-26 19:58:19.030000000 +0000
***************
*** 22,28 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 0|S2|3O3|5|40|40|1
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 22,28 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 0|S2|3O3|5|0|0.0|1
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
***************
*** 48,54 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: S2|Ricardo Figueiredo|Lisboa|NORMAL|40|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 48,54 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
