*** auto-tests/A-16-10-M-ok.out	2020-11-26 19:26:04.100000000 +0000
--- auto-tests/A-16-10-M-ok.outhyp	2020-11-26 19:58:19.270000000 +0000
***************
*** 16,22 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto '6M2': pedido=10000, existências=100
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 16,22 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto 'S2': pedido=10000, existências=100
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
