*** auto-tests/A-18-05-M-ok.out	2020-11-27 17:15:35.266804200 +0000
--- auto-tests/A-18-05-M-ok.outhyp	2020-11-27 19:15:58.755448400 +0000
***************
*** 163,169 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|SELECTION|206|206
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 163,169 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|206|206
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 171,177 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|SELECTION|206|206
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
--- 171,177 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|206|206
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
