*** auto-tests/A-18-12-M-ok.out	2020-11-29 18:29:14.070000000 +0000
--- auto-tests/A-18-12-M-ok.outhyp	2020-11-29 18:33:17.390000000 +0000
***************
*** 170,176 ****
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 170,177 ----
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Guardar: Operação inválida: Problema ao abrir 'selection.woo'.
! Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
